.github/workflows/run-build-and-acceptance-tests.yml

```
@@ -213,23 +227,62 @@ jobs:
         with:
           repo: t0yv0/goteststats
           tag: v0.0.7
-      - name: Ensure
+      - name: Download Pulumi Go Binaries (linux-x64)
+        if: ${{ matrix.platform == 'ubuntu-latest' }}
+        uses: actions/download-artifact@v2
+        with:
+          name: pulumi-linux-x64
+          path: artifacts/go
+      - name: Download Pulumi Go Binaries (darwin-x64)
+        if: ${{ matrix.platform == 'macos-latest' }}
+        uses: actions/download-artifact@v2
+        with:
+          name: pulumi-darwin-x64
+          path: artifacts/go
+      - name: Install Pulumi Go Binaries
         run: |
-          make ensure
-      - name: Dist
+          mkdir -p bin
+          tar -xf artifacts/go/*.tar.gz -C bin
+          echo "$PWD/bin/pulumi" >> $GITHUB_PATH
+      - name: Download Pulumi .NET NuGet packages
+        uses: actions/download-artifact@v2
+        with:
+          name: pulumi-nuget-packages
+          # path set to match PULUMI_LOCAL_NUGET
+          path: ${{ runner.temp }}/opt/pulumi/nuget
```

Can you set this directly with the env var?

```suggestion
          path: ${{ runner.temp }}${{ env.PULUMI_LOCAL_NUGET }}
```

---

That wouldn't work since PULUMI_LOCAL_NUGET is not set at a workflow level but set step-level locally only; leaving as is for now, I'm revisiting this anyway in the windows PR.